lib/lib.misc.c

```diff
@@ -28,19 +28,19 @@ void _alert(const char *libdir, const char *msgtype, const char *description)
 {
 	static char none[1] = "";
 	char *cmd;
-	int dummy __attribute__((unused));
+	int temp __attribute__((unused));
```

Comment:
`temp` is not correct here, because it implies the result is used.

Reply:
Do you prefer silenced or unchecked in this case? I think the meaning is also clear already from the comments. In this case I went for silenced (as thats whats happening) and the reasoning why this is silenced is clear from the comments.